tooling-tests/src/test/resources/test-src/input/tool_test_init_1/Ballerina.toml

```diff
@@ -6,3 +6,8 @@ distribution = "2201.2.0"
 
 [build-options]
 observabilityIncluded = true
+
```

Comment:
We need to update the same in output resources right?

Reply:
it's already there, since we are testing the case where dependancy is already there in the ballerina.toml file